src/View/TwigView.php

```diff
@@ -258,12 +258,13 @@ protected function initializeProfiler(): void
      */
     protected function _render(string $templateFile, array $data = []): string
     {
-        $viewVar = $this->getConfig('viewVar');
+        $contextName = $this->getConfig('contextName');
         $data = array_merge(
             empty($data) ? $this->viewVars : $data,
             iterator_to_array($this->helpers()->getIterator()),
             [
-                $viewVar => $this,
+                '__internal' => $this,
+                $contextName => $this,
```

Comment:
I wish we could avoid the duplicate assignment, but don't have any ideas right now for it. Also we should probably use `addGloba()` to set the view instance instead of merging it with regular vars https://twig.symfony.com/doc/3.x/advanced.html#globals

Reply:
The only alternative I could figure out was assigning the variable name to something, reading that and then using that as the context index. But that's silly and slower just to avoid double assignment.

Reply:
So, one difference between globals and context is whether macros have access to them by default.

I think it makes sense for this to be available to macros by default.